integrations/boost_build.rst

```diff
@@ -2,6 +2,11 @@
 |boost_build_logo| Boost Build
 ______________________________
 
+.. container:: out_reference_box
```

Comment:
Don't know if this is a highlighted box in restructuredtext. Maybe use ``.. important::`` ?